CHANGELOG.md

```diff
@@ -24,6 +27,15 @@
 
   ...though the exact form differs by dialect.
 
+- Change to function syntax: `let f = x -> ...` See
+  https://github.com/PRQL/prql/blob/main/web/book/src/queries/functions.md
```

Comment:
```suggestion
  <https://github.com/PRQL/prql/blob/main/web/book/src/queries/functions.md>
```

Note that links generally need to be enclosed in `<>`.

Reply:
- It's OK to just have one liners here as reminders for someone else to do it
- Ultimately these should be formatted links to the book like [Functions docs](https://prql-lang.org/book/queries/functions.html)
- To the extent we can do that way initially, that's ideal!